src/main/java/sirius/biz/jobs/scheduler/SchedulerController.java

```diff
@@ -103,6 +120,53 @@ public void schedulerEntry(WebContext webContext, String entryId) {
         }
     }
 
+    /**
+     * Executes the given scheduler entry immediately and redirects to the process detail view.
+     *
+     * @param webContext the current request
+     * @param entryId    the id of the entry to display or <tt>new</tt> to create a new one
+     */
+    @Permission(PERMISSION_MANAGE_SCHEDULER)
```

Comment:
I would change to PERMISSION_EXECUTE_JOBS instead. In 99% of the cases the user has both, but execution is a bit more appropriate imo.